src/pages/communities/[slug]/index.tsx

```diff
@@ -22,18 +24,27 @@ export default function Slug(props: {
   const { community, challenges } = props.pageProps;
   const { t } = useTranslation();
   return (
-    <CommunityWrapper>
-      {challenges.map((challenge) => (
-        <ChallengeCard key={challenge.id} data={challenge} community={community} />
-      ))}
-      <div className="md:hidden">
-        <div className="active md:hidden mb-7 scroll-smooth pt-5">
-          <div className="font-medium text-.5xl leading-snug">{t("communities.overview.scoreboard.title")}</div>
-          <div className="text-sm font-light lg:w-full lg:pr-7 pt-2">{t("communities.overview.scoreboard.description")} </div>
+    <>
+      <Head>
+        <title>{getMetadataTitle(t("communities.navigation.courses"), community?.name as string)}</title>
```

Comment:
can you make the page title as a component which will then handle everything within itself